internal/fixture/state.go

```diff
@@ -146,7 +147,8 @@ func (s *State) ApplyObject(tv *typed.TypedValue, version fieldpath.APIVersion,
 	if err != nil {
 		return err
 	}
-	new, managers, err := s.Updater.Apply(s.Live, tv, version, s.Managers, ignored, manager, force)
+	defer setIgnoredFieldsAndCleanup(s.Updater, ignored)()
```

Comment:
Isn't this wrong? You're trying to move something that per-operation to something that should be a state? This should be a parameter of the `Test` method.

Reply:
Yes, but in the current test cases i wanted to be able to change the ignoredFields per operation.
Aside from that i had the same thought.
Mainly the `update_does_not_steal_ignored` test is the reason i change the ignoredFields to emulate behavior like the one of a subresource (where they differ per operation).